clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/common/AzureUtilities.groovy

```diff
@@ -112,6 +113,30 @@ class AzureUtilities {
     resourceId.split(PATH_SEPARATOR).last()
   }
 
+  // id = "/subscriptions/***-***-***/resourceGroups/***/providers/Microsoft.Network/networkInterfaces/nic1/ipConfigurations/ipconfig1"
+  static String getResourceNameFromId(String id) {
```

Comment:
Why will we need to get the resource name from a different provider? This is in AzureUtilities... I can stick "Azure" in the method name if we care about how precisely we should be here.

Thanks,
Milis

Sent from my Windows Phone
________________________________
From: Scot Moorhead<mailto:notifications@github.com>
Sent: ‎9/‎6/‎2016 10:45 AM
To: Microsoft/clouddriver<mailto:clouddriver@noreply.github.com>
Cc: Milis Militaru<mailto:adrianmi@microsoft.com>; Author<mailto:author@noreply.github.com>
Subject: Re: [Microsoft/clouddriver] Add support for selecting an existing virtual network and subnet (#38)


In clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/common/AzureUtilities.groovy<https://github.com/Microsoft/clouddriver/pull/38#discussion_r77683956>:

> @@ -112,6 +113,30 @@ class AzureUtilities {
>      resourceId.split(PATH_SEPARATOR).last()
>    }
>
> +  // id = "/subscriptions/***-***-***/resourceGroups/***/providers/Microsoft.Network/networkInterfaces/nic1/ipConfigurations/ipconfig1"
> +  static String getResourceNameFromId(String id) {


This method needs to be generic and not hard-coded to just a single provider. What if we need to get the name of a resource that's from a different provider?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/clouddriver/pull/38/files/3483d73d258bd392d617bf547b8853a662b52b6b#r77683956>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOdiXGD8R6j9jL564O9RAUupi0Q1NI3zks5qnaaugaJpZM4Jz9MG>.